arches/app/media/css/arches.scss

```diff
@@ -5386,144 +5386,7 @@ div.dropdown-menu.open {
     position: absolute;
 }
 
```

Comment:
there are additional password css rules that should be moved over to the _authentication.scss file

```
.password-rules {
    margin-top: -5px;
    font-size: 12px;
    color: #555;
}

.password-rule {
    color: #888;
    font-size: 13px;
    padding-left: 5px;
    padding-bottom: 3px;
}

.password-rule span {
    padding-left: 3px;
    font-size: 12px;
}

.password-rule i {
    font-size: 11px;
}
```

Also can you make sure that these values render correctly in RTL. 

<img width="506" alt="Screenshot 2023-04-14 at 5 59 51 PM" src="https://user-images.githubusercontent.com/823773/232175449-04c8092c-a97e-4083-ae6a-56a4076a86fb.png">